### PR TITLE
Added details about services requirements

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
@cadecairos I added as much detail as possible based on what we know from design/product, I was hoping you could look it over and start to make some technical decisions with @simonwex 

cc @thisandagain @xmatthewx

File: README.md [hunk 1]
```diff
@@ -1,2 +1,110 @@
 # api.webmaker.org
-Services for Webmaker
+
+This is where we'll build the service for saving, publishing, and searching projects created by the Webmaker Android application.
```
Comment (cadecairos) [inline README.md]: Isn't the long-term goal to have more than just the android app consume this service?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,110 @@
...
+  - have **privileged metadata**, such as featured or banned status, that is only editable by admin level accounts
+- if possible, `Project`s should be able to be migrated based on their version numbers.
+- a more specific definition of this data model needs to be determined once design is finalized.
+
+## API Requirements
+
+### Projects API
+
+Basic needs: retrieve and edit projects belonging to a user
+Authentication: unauthed: no acess; authed: read-write; admin: read-write
```
Comment (cadecairos) [inline README.md]: acess -> access

File: README.md [hunk 3]
```diff
@@ -1,2 +1,110 @@
...
+
+### Create/Read/Update/Delete projects
+- create a new `Project`, return an ID
+- read/update/delete a `Project` by ID
+- should implement "soft delete"
```
Comment (cadecairos) [inline README.md]: soft delete is not fun. do we really need this?

**Comment by cadecairos** [comment]
@k88hudson thanks, just left a few things to look at, then I'd say merge it.

File: README.md (hunk 1 above)
Comment (k88hudson) [inline README.md]: yes, good point!

File: README.md (hunk 3 above)
Comment (k88hudson) [inline README.md]: hm. we need to have a way to undo deletions, but there are other ways to handle this. happy to leave it up for discussion

**Pull request merged**